src/internalBinding/timers.cc

```diff
@@ -44,13 +44,13 @@ static bool cancelByTimeoutId(JSContext *cx, unsigned argc, JS::Value *vp) {
 
   args.rval().setUndefined();
 
-  // silently does nothing when an invalid timeoutID is passed in
-  if (!timeoutIdArg.isInt32()) {
+  // silently does nothing when an invalid timeoutID (should be an int32 value) is passed in
```

Comment:
does the TODO on line 39 need an update?

Reply:
Replaced by https://github.com/Distributive-Network/PythonMonkey/pull/236/commits/fa94dae30878c343718903aef7f6d27097ab2487